tests/braille-specs/Makefile.am

```diff
@@ -65,7 +65,6 @@ EXTRA_DIST =					\
 	hr-8dots_harness.yaml			\
 	hu-hu-comp8_backward.yaml		\
 	hu-hu-comp8_harness.yaml		\
-	hu-hu-g1-hyph_harness.yaml		\
```

Comment:
Since you moved the yaml test shouldn't it be added to `Makefile.am` in `test/yaml/Makefile.am`

Reply:
I thought I did. But apparently not.

I wish there was a way to not have to do this Makefile.am manipulation all the time. I've never understood why you have to have a Makefile.am with EXTRA_DIST for every subdir. Couldn't this be generated? Especially with the liblouisutdml test suite it is getting absurd. For every new test you even have to add something to configure.ac. Why? Isn't this information already present in the SUBDIRS variables?